src/client.js

```diff
@@ -9,11 +9,11 @@ import ExtJSON from 'mongodb-extjson';
 import queryString from 'query-string';
 import {StitchError, ErrInvalidSession, ErrUnauthorized} from './errors';
 
-const v1 = 1;
+export const v1 = 1;
 const v2 = 2;
-const v3 = 3;
+export const v3 = 3;
 const API_TYPE_PUBLIC = 'public';
-const API_TYPE_PRIVATE = 'private';
+export const API_TYPE_PRIVATE = 'private';
```

Comment:
[nit] alternatively you could move these to a constants file that both can import from